project/AddMetaInfLicenseFiles.scala

```diff
@@ -41,6 +41,13 @@ object AddMetaInfLicenseFiles extends AutoPlugin {
     apacheSonatypeLicenseFile := baseDir.value / "LICENSE",
     apacheSonatypeNoticeFile := baseDir.value / "NOTICE")
 
+  /**
+   * Settings specific for Pekko http-cors subproject which require a different license file.
+   */
+  lazy val httpCorsSettings = Seq(
+    apacheSonatypeLicenseFile := baseDir.value / "legal" / "StandardLicense.txt",
+    apacheSonatypeNoticeFile := baseDir.value / "legal" / "CorsNotice.txt")
```

Comment:
does it need a `DISCLAIMER` too?

Reply:
this is inherited from the projectSetting - that adds the DISCLAIMER to all jars - it is the same DISCLAIMER in all jars but the LICENSE and NOTICE can be specific to a jar

Reply:
Yes, by default https://github.com/mdedetrich/sbt-apache-sonatype will add the DISCLAIMER into `projectSettings` since its a mandatory for ASF projects.